helper/ResponseVariableFormatter.php

```diff
@@ -179,4 +178,13 @@ public static function formatStructuredVariablesToItemState(
 
         return $formatted;
     }
+
+    private static function formatVariableFile(ResponseVariable $responseVariable): array
```

Comment:
May we add some test?

Reply:
I'd like to do so but I spend a few hours trying to set up everything and could not make it. Considering the changes, it should be alright.